annotations/2.0/_index.md

```diff
@@ -1,51 +1,23 @@
 ---
-title: "Jakarta Annotations 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta Annotations (milestone)"
+date: 2020-05-31
 summary: "Release for Jakarta EE 9"
 ---
-Jakarta Annotations defines a collection of annotations representing common semantic concepts that
-enable a declarative style of programming that applies across a variety of Java technologies.
 
+Jakarta Annotations defines a collection of annotations representing common semantic concepts that enable a declarative style of programming that applies across a variety of Java technologies.
+
+* [Jakarta Annotations 2.0 Release Record]()
 * [Jakarta Annotations 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.ca/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Annotations 2.0 Specification Document]() (PDF)
-* [Jakarta Annotations 2.0 Specification Document]() (HTML)
+* [Jakarta Annotations 2.0 Specification Document](./annotations-spec-2.0-RC1.pdf) (PDF)
+* [Jakarta Annotations 2.0 Specification Document](./annotations-spec-2.0-RC1.html) (HTML)
 * [Jakarta Annotations 2.0 Javadoc](./apidocs)
 * [Jakarta Annotations 2.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
-  * [jakarta.annotation:jakarta.annotation-api:jar:2.0.0]()
-
+  * [jakarta.annotation:jakarta.annotation-api:jar:2.0.0-RC1](https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/2.0.0-RC1/jar)
 
 # Compatible Implementations
 
-* [Jakarta Annotations]()
+* [Jakarta Annotations](https://eclipse-ee4j.github.io/common-annotations-api/)
 
 # Ballots
-
```

Comment:
Do not delete this material.  This is all good material for the Release Plan Review results.